wiki/en/Server-Install.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: wiki
+title: "Installing a Server"
+lang: "en"
+permalink: "/wiki/Server-Install"
+---
+
+{% include breadcrumb.html root="Server Guide" branch1="Server Manual" branch1-url="Running-a-Server" %}
+
+# Installing a Server
+
+Make sure you read the [Server Manual](Running-a-Server).
+
+Most people run Jamulus as a "pure" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following guide assumes you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop).  
+
+If you want to run a Server on a Raspberry Pi, you will need to [compile from source](https://github.com/jamulussoftware/jamulus/blob/master/COMPILING.md). See also this [guide for Raspberry Pi](/kb/2020/03/28/Server-Rpi.html) maintained by Jamulus user fredsiva.
+
+### Installation
+
+1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
+1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
+1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Reload the systemd files `sudo systemctl daemon-reload` and restart the headless Server: `sudo systemctl restart jamulus-headless`
+1. Check all is well with `systemctl status jamulus-headless` (hit `q` to get back to the command prompt).
+
+**Note**: You can upgrade your server by repeating the steps mentioned above.
+
+### Configuration
+
+You can control Jamulus with the `systemctl` command. For example, to stop the Server cleanly:
+
+`sudo systemctl stop jamulus-headless`
+
+### Running an Unregistered Server
+
+It is highly recommended to test your Server on a **Public Directory** first so as to narrow down any subsequent problems in unregistered mode.
```

Comment:
In which case, move the registered section before the unregistered section..?

Reply:
Yes, I did that since I thought most people reading that would still want to install a unregistered server. I think we should drop this (or move it to the GUI section).